package.json

```diff
@@ -25,10 +25,6 @@
     "heroku-postbuild": "patch-package && npm run build"
   },
   "dependencies": {
-    "@testing-library/jest-dom": "^5.14.1",
```

Comment:
I moved these frontend dependencies to `devDependencies` section, since they are only needed during the build process. So all packages in `dependencies` are only reserved for the backend, which is required in runtime.